docs/user-guide/Opening-a-Translation-Project.md

```diff
@@ -0,0 +1,2 @@
+# Instructions on opening a translation project
+
```

Comment:
Needs more words.

For these sections that don't have their content yet, we could quickly add the URL for the current content. So that:

 - We have an easy reference when we are ready to add the content.
 - If we miss adding content for some reason, the user gets more than an empty page.